src/test/java/com/verhas/licensor/TestLicenseClass.java

```diff
@@ -53,6 +53,14 @@ public void tearDown() throws IOException {
         new File(dumpFile2).delete();
     }
 
+    private int getOffset() {
```

Comment:
Bad naming style. Why `get`? It is not a JavaBean to be forced by some legacy tools to start with `get`. This is an offset, which is also something not really talkative. I would recommend something like `numberOfLineSeparatorCharacters()` or `nrLineSepChars()` if the first is too long for your taste.